.changeset/mean-candles-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Fix auth event uncancellable bug
```

Comment:
nit: include reference to https://github.com/firebase/firebase-js-sdk/issues/7383

Reply:
I added issue number